flake.nix

```
@@ -537,6 +540,27 @@
             FONTCONFIG_FILE = texFunctions.fontsConf;
           };
 
+        # API docs for unstable C++ interfaces.
+        api-docs =
```

```suggestion
        internal-api-docs =
```

It didn't reflect the intent sufficiently.